code/datums/achievements/misc_achievements.dm

```diff
@@ -84,11 +84,11 @@
 	desc = "How does it feel to know that your workplace values a mop bucket on wheels more than you?" // i can do better than this give me time
 	database_id = MEDAL_CLEANBOSS
 
-/datum/award/achievement/misc/rule8
-	name = "Rule 8"
+/datum/award/achievement/misc/rule3
+	name = "Rule 3"
```

Comment:
Technically all you have to do is change the name, to avoid further merge conflicts.

Reply:
So you're saying to leave everything but the name as rule 8?

Reply:
Yep. It doesn't matter to players what's in the code.

Reply:
Alright